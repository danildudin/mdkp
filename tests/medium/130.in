30 10 0
371 589 899 498 389 112 233 269 4 938 97 658 627 57 915 319 372 916 543 642 781 968 974 560 729 927 956 882 922 528 
880 261 753 776 365 77 589 23 368 977 497 306 214 556 956 703 698 173 625 725 339 470 470 300 811 128 607 880 348 827 
443 663 372 796 398 410 226 64 380 504 666 803 670 312 436 932 775 448 376 176 865 936 757 532 771 358 74 780 237 475 
449 826 336 172 867 84 948 138 97 164 605 162 595 369 104 736 156 757 137 597 480 68 391 913 378 342 450 13 254 332 
72 60 498 490 507 23 880 964 985 449 106 457 677 46 442 43 631 948 479 770 809 769 897 955 703 335 194 84 679 798 
572 934 407 530 834 817 408 234 664 716 408 660 601 58 750 675 673 124 178 703 238 635 871 484 924 561 616 201 793 156 
291 875 786 73 531 957 190 313 438 346 528 392 499 37 708 830 340 789 990 463 308 720 495 468 218 340 435 631 831 267 
328 387 725 367 252 426 717 564 484 331 450 495 652 827 550 526 834 754 254 1 891 573 867 582 737 982 724 380 401 299 
768 95 986 689 115 632 805 321 176 623 400 992 26 413 982 869 105 781 471 677 152 397 844 702 585 648 233 909 738 136 
34 277 40 743 736 251 990 318 662 868 23 656 596 502 358 913 582 368 78 584 221 36 615 348 448 633 341 779 780 431 
566 859 798 184 791 482 346 104 112 138 34 482 990 256 88 484 150 94 65 79 300 326 855 943 364 529 957 963 384 124 
6259 5388 8421 5390 5795 9610 6638 6728 9270 8203 
