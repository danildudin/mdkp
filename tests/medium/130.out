9268
11
1 2 9 14 17 18 20 25 26 27 28 
